### PR TITLE
Overall eval analyzing metrics

### PR DESCRIPTION
**Pull request by hinckleyc**
Finished our part of the demo for the lab. Please let me know if we need to fix anything.

### CODE REVIEW

**Comment by spitalnyd** [comment]
> Finished our part of the demo for the lab. Please let me know if we need to fix anything.

I think in the future it would be useful to have project managers and technical leaders in the review  as well.

**Comment by bagashvilit** [comment]
I would encourage you to have a more informative description of the pull request, for example, include the reason why the pull request was made, mention the related issues you are facing, or describe the changes made, if applicable.

File: calculateMetrics/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+"""Empty file required by pylint."""
```
Comment (enpuyou) [inline calculateMetrics/__init__.py]: I don't think `_init_.py` is just to be there for pylint. Maybe do some research on what it actually does. It is a minor issue, though, since it's only a module docstring

File: calculateMetrics/metrics.py [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+    added_scores = []
+    username_accesser = 0
+    list_length = len(github_data)
+    # for ab in range(len(github_data)):
+    for cd in range(len(github_data)):
+        print("Checking GitHub user: ", github_data[username_accesser][0])
+        # This if/else calculates the amount added to the standard deviation,
+        # in order to grade the team.
+        # This if statment will end if the user was within 0.5 of the standard deviation.
+        if github_data[username_accesser][2] <= np.average(added_list) + (
```
Comment (enpuyou) [inline calculateMetrics/metrics.py]: Since `github_data[username_accesser][2]` and `np.average(added_list)` are being used all over the place, do you think if it would be a good idea to store their values into variables and just call the variables?

File: calculateMetrics/metrics.py [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+    removed_sd = standard_deviations_list[2]
+    removed_scores = []
+    username_accesser = 0
+    list_length = len(github_data)
+    for ef in range(len(github_data)):
+        print("Checking GitHub user: ", github_data[username_accesser][0])
+        # This if/else calculates the amount of code removed to the,
+        # standard deviation in order to grade the team.
+        #Ths if statment will end if the user was within 0.5 of the standard deviation.
+        if github_data[username_accesser][3] <= np.average(removed_list) + (
```
Comment (enpuyou) [inline calculateMetrics/metrics.py]: Same here with the use of `github_data[username_accesser][3]` and `np.average(added_list)`

File: calculateMetrics/metrics.py [hunk 4]
```diff
@@ -0,0 +1,381 @@
...
+            username_accesser += 1
+        else:
+            print("Appending 0 to removed_scores list")
+            removed_scores.append(0)
+            username_accesser += 1
+    global removed_overall_score
+    removed_overall_score = np.average(removed_scores)
+    # These print statments print out the scores for the amount of code the team,
+    # removed.
+    print("")
```
Comment (enpuyou) [inline calculateMetrics/metrics.py]: I would suggest using ` \n` instead of a blank print statement to start a new line

File: calculateMetrics/metrics.py [hunk 5]
```diff
@@ -0,0 +1,452 @@
...
+    ["johnSmith", 22, 349, 50, 4],
+    ["janeDoe", 23, 375, 30, 3],
+    ["Edgar_AllenPoe", 27, 315, 75, 7],
+    ["George_Washington", 25, 360, 65, 10],
+    ["Alexander_Hamilton", 41, 530, 100, 230],
+    ["Karl_Marx", 0, 0, 0, 400],
+    ["Julius_Caesar", 25, 310, 68, 1],
+    ["Napoleon_Bonaparte", 24, 363, 70, 3],
+    ["Alexander_the_Great", 42, 540, 110, 1],
+]
```
Comment (MaddyKapfhammer) [inline calculateMetrics/metrics.py]: When will this code be modified to use data that has been retrieved from GitHub rather than fake data? I believe that this will cause some issues, but is essential to merging this PR.

File: calculateMetrics/old_metrics.py [hunk 6]
```diff
@@ -0,0 +1,319 @@
+"""Determine how the well the team worked together"""
```
Comment (MaddyKapfhammer) [inline calculateMetrics/old_metrics.py]: Is this file required? As it seems that the above file `metrics.py` does these calculations and more. If it is not necessary, I would suggest it being deleted.

**Comment by bagashvilit** [comment]
I am working on Individual scoring and the fake data we use is in a nested dictionary, where the username and the individual categories such as: `COMMITS`,`ADDED`, `REMOVED` are the `key`. Please see the [link](https://github.com/GatorCogitate/cogitate_tool/blob/Individual-scoring/src/Individual_score.py)I think we should discuss this tomorrow to make sure we are all on the same page.

**Comment by hinckleyc** [comment]
Should we close this pr since we are not using this overall eval analyzing metric?
